.github/workflows/ci.yaml

```diff
@@ -8,4 +8,4 @@ jobs:
   ci:
     uses: nextstrain/.github/.github/workflows/pathogen-repo-ci.yaml@master
     with:
-      build-args: auspice/flu_avian_h5n1_ha.json
```

Comment:
For future considerations of maintaining X number of pathogen repos, instead of using an exact file target, we could add a stable rule target.

E.g. add a `pathogen_ci` rule that points to the expected output:
```
rule pathogen_ci:
    input: "auspice/avian-flu_h5n1_ha.json"
```

Then the CI build args can always be
```
build-args: pathogen_ci
```

This way any future changes to the output file paths can be done within individual pathogen repos instead of needing to update CI workflows across all our tools.

Reply:
Good idea!

Reply:
Continuing in #17